### DISCUSSION TITLE
How to timeout session?

### DISCUSSION THREAD

**Discussion by poonai** [pomerium/pomerium#2026] [Q&A] [answered]
Hey,

I'm using GitHub of idp_provider and I want to terminate the authorized session after a specified time. 

Couldn't able to find it on docs.

Btw, your slack invite server not working.

**Comment by desimone** [comment, marked as answer]
Hey @captain-me0w; 

See [cookie_expire](https://master.docs.pomerium.io/reference/#cookie-options) under cookie options. That will set both the maximum client side (cookie) expiration, and backend session expiration period. 

> Btw, your slack invite server not working.

Thanks for the heads up; it looks like our invite bot broke because of an API change in slack.  The following link should work until we find a decent workaround. 

https://join.slack.com/t/pomerium-io/shared_invite/zt-o2ith4qr-_qYANnwpW09p1R_~fYPP4w

https://github.com/pomerium/pomerium/issues/2014#issuecomment-804198572

**Reply by poonai** [reply]
Thanks :)
